src/queries.jl

```diff
@@ -223,6 +224,40 @@ function isdeterministic(root::LogicCircuit)::Bool
     result
 end
 
+
```

Comment:
I guess you could also add a doc string to describe input/outputs and what implied_literals means a bit.